app/src/components/title.tsx

```diff
@@ -0,0 +1,28 @@
+export default function Title({
+  text,
+  dark = true,
+  textSize = "text-2xl",
+}: {
+  text: string;
+  dark?: boolean;
+  textSize?: string;
+}) {
+  var linesClassNames = "w-10 h-0.5 " + (dark ? "bg-blue-950" : "bg-white");
```

Comment:
Use `const` instead of `var` whenever possible. Also prefer using string interpolation, i.e:

```suggestion
  const linesClassNames = `w-10 h-0.5 ${dark ? "bg-blue-950" : "bg-white"}`;
```